src/components/AccordionPanel/AccordionPanel.tsx

```diff
@@ -9,6 +9,7 @@ import React, {
   ReactNode,
 } from 'react';
 import styles from './AccordionPanel.module.css';
+import { EdsThemeColorIconNeutralStrong } from '../../tokens-dist/colors';
```

Comment:
I double-checked that this gets included when we build to publish. It is a little weird to import from `tokens-dist` internally though (external imports will use `dist/tokens`) 🤔  Alternatively, we could rename tokens-dist -> tokens but then we have to live with a "design-tokens" folder and a "tokens" folder...